content/java/concepts/hashmap/hashmap.md

```diff
@@ -181,3 +181,35 @@ Jeanette
 Ben
 Lily
 ```
+
+
+##Traversing a `HashMap` using **map.entrySet()**
+
+Alternatively, we could use Map.entry() to return a `set` of key-value pairs. This might be more efficient is some cases, where we might need to access the key and the value.
```

Comment:
```suggestion
Alternatively, `.entrySet()` can be used to return a `set` of key-value pairs. This might be more efficient is some cases, where it's needed to access the key and the value.

```
We avoid first- and second-person pronouns in Docs.

Reply:
Thanks for the feedback. I changed it. 😃